cmd/minikube/cmd/config/validations.go

```diff
@@ -59,6 +58,35 @@ func IsValidURL(name string, location string) error {
 	return nil
 }
 
+func IsURLExists(name string, location string) error {
+	parsed, err := url.Parse(location)
+	if err != nil {
+		return fmt.Errorf("%s is not a valid URL", location)
```

Comment:
This duplicates the code in `IsValidURL(string, string) error` (which can be found above my function). I had three choices:
* Combine this code into "IsValidURL" (opted not to because a URL can be valid, syntax-wise, without describing a resource that exists)
* Ignore the error (assume this function is run after IsValidURL)
* Duplicate the check

I thought the third option would be best if this function were to be re-used in the future. I am more than happy to do the first option, though, and perhaps change the name of `IsValidURL` to something like `IsRealURL`.